### PR TITLE
#167749939 Set up sequelize models and migrations

### PR DESCRIPTION
**Pull request by Naimaoye**
#### What does this PR do?
set up users models and migrations using sequelize
#### Description of Tasks to be completed?
add sequelize associates and other models and migrations
#### How should this be manually tested?
run `npm db:migrate` to create tables in the database
#### What are the relevant pivotal tracker stories?
[167749939](https://www.pivotaltracker.com/n/projects/2381955)

### CODE REVIEW

**Review by davidteejay — CHANGES_REQUESTED** [review]
I think all you did should be in ES6. I see some of your code is, but all of them should be.

**Comment by Naimaoye** [comment]
Thanks for the feedback david. I dont know what is wrong with sequelize, I tried using es6 but it wasnt working and was throwing error. I guess they haven't updated it to es6 format

**Review by davidteejay — APPROVED** [review]

File: .sequelizerc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const path  = require('path')
+module.exports = {
+    "config": path.resolve('./src/database/config', 'config.js'),
+    "models-path": path.resolve('./src/database/models'),
+    "seeders-path": path.resolve('./src/database/seeders'),
+    "migrations-path": path.resolve('./src/database/migrations')
+};
```
Comment (hustlaviola) [inline .sequelizerc]: You can always add a `new line` at the end of a file, one of `eslint` rules. Good work

**Review by hustlaviola — COMMENTED** [review]
The instruction to install `sequelize-cli` globally on `README.md` is redundant since `sequelize-cli` is in `package.json`, you can remove it.
Once `es6 codebase` PR gets merged, you can convert the codebase to es6. The ones that throw errors will be added to `.eslintIgnore`. Good job on this PR.

**Review by hustlaviola — COMMENTED** [review]
You can pluralize this:
Screenshot (116)
You can also change this to `run`

Screenshot (117)

**Review by davidteejay — APPROVED** [review]

**Review by hustlaviola — CHANGES_REQUESTED** [review]
Good job. You can edit the title of the PR.  Follow this format `#167749948 Deploy base application to Heroku`

**Review by davidteejay — APPROVED** [review]

**Review by hustlaviola — APPROVED** [review]

File: .sequelizerc (hunk 1 above)
Comment (ascii-dev) [reply .sequelizerc]: @Naimaoye please add a new line here. Thanks

**Review by davidteejay — CHANGES_REQUESTED** [review]
The Model you wrote isn't written correctly. Also, you haven't written the database connection yet. Please go through the sequelize documentation here: https://sequelize.org/master/manual/getting-started.html

**Review by davidteejay — APPROVED** [review]

File: src/database/config/config.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+require('dotenv').config();
```
Comment (ldonjibson) [inline src/database/config/config.js:1]: replace with es6 .. use **import**

File: .sequelizerc [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const path  = require('path')
```
Comment (ldonjibson) [inline .sequelizerc:1]: replace with es6 . **import**

**Review by ldonjibson — CHANGES_REQUESTED** [review]

**Review by ldonjibson — APPROVED** [review]
Weldone naimat good job!

**Review by ascii-dev — APPROVED** [review]
LGTM

**Pull request merged**